handlers/mesh.go

```diff
@@ -46,6 +49,7 @@ func MeshGraph(
 		o := mesh.NewOptions(r, &business.Namespace)
 
 		meshInfo, err := discovery.Mesh(r.Context())
+		filterAccessibleControlPlanes(r.Context(), business.Namespace, meshInfo)
```

Comment:
I'm not sure if we should filter out the controlplanes the user doesn't have access to entirely or if we should show them as inaccessible nodes like what we do with the traffic graph.

Reply:
> I'm not sure if we should filter out the controlplanes the user doesn't have access to entirely or if we should show them as inaccessible nodes like what we do with the traffic graph.

Maybe for security reasons, if the user doesn't have access to the cluster it would be better to just hide it?

Reply:
I see in the Traffic graph the user can see the inaccessible nodes: 

![image](https://github.com/user-attachments/assets/6c05304b-9645-4028-87a9-581f013cb2b4)

But not sure if that should be the same for the Mesh.

Reply:
I'm fine with either one. I think if the user didn't have access they would still see the cluster node wouldn't they? It just wouldn't have the controlplane node in there?